src/components/reviews/reviewTile.jsx

```diff
@@ -27,9 +27,9 @@ class ReviewTile extends React.Component {
     return (
       <div className="tile is-child box">
         <div>{review.rating}-star rating goes here</div>
-        {this.state.verified ?
-          <div className="has-text-right is-size-6">{review.reviewer_name}, {moment(review.date, moment.ISO_8601).format("MMMM Do YYYY")}</div>
-          : <div className="has-text-right is-size-7">{review.reviewer_name}, {review.date}</div>}
+        {this.state.verified
+          ? <div className="has-text-right is-size-7">{review.reviewer_name}, {moment(review.date).format("MMMM DD, YYYY")}</div>
```

Comment:
I am interested in the moment library. We will have to walk through simple usage at some point. Again, its interface makes things easier to read here.